src/app/qgsoptions.cpp

```diff
@@ -2300,3 +2300,25 @@ void QgsOptions::scaleItemChanged( QListWidgetItem *changedScaleItem )
   addScaleToScaleList( changedScaleItem );
   mListGlobalScales->setCurrentItem( changedScaleItem );
 }
+
+double QgsOptions::getZoomFactorValue(QSpinBox *zoomFactorSpinBox)
+{
+  // Get the decimal value for zoom factor. This function is needed because the zoom factor spin box is shown as a percent value.
+  // The minimum zoom factor value is 1.01
+  if ( zoomFactorSpinBox->value() == zoomFactorSpinBox->minimum() ) return 1.01;
+  else return zoomFactorSpinBox->value()/100.0;
+}
+
+void QgsOptions::setZoomFactorValue(QSpinBox *zoomFactorSpinBox)
```

Comment:
The spacing is wrong here. You can setup a pre-commit hook as described here https://github.com/qgis/QGIS/blob/master/CONTRIBUTE.md so it's done automatically for you.